login.css

```diff
@@ -22,7 +22,7 @@ body{
 
 nav{
 	width:100%;
-	background-color:white;
+	background-color:#C8A2C8;
```

Comment:
05/08/2021: I trailing to apply the color combination that I decided and display it in the browser to see if it easy to read and make my music database to be aesthetic. I change the background color of the nav to to lilac.